README.md

```diff
@@ -150,7 +133,71 @@ npm run build
+- Check out the current supported branch of the AAOS Unbundled repo (see the "Check out the Unbundled AAOS Repo" job in `.github/workflows/main` for the correct branch) and set $ORG_GRADLE_PROJECT_unbundledAAOSDir to the path of the checkout
+- Set `$FIGMA_ACCESS_TOKEN` to your Figma token
+
+The test-all script takes an optional `-s` flag to skip all emulator tests. It's intended for situations where emulators can't be started or when running tests before updating serialized files.
+
+## Before you submit for review:
+
+To check that you can pass presubmits and emulator tests before pushing to PR you can do the following:
+
+  ```shell
+  ./dev-scripts/clean-all.sh #Optional, but will ensure you're testing a clean environment.
+  ./dev-scripts/format-all.sh
+  ./dev-scripts/test-all.sh
+  ```
+
+## Release process
+
+1. Create a new release in GitHub, set the tag to "v\<version>" (see previous releases for examples). This will trigger a release build action. \
+        You can watch it's progress [here](https://github.com/google/automotive-design-compose/actions/workflows/release.yml).
+2. Once complete the release artifacts will be uploaded to the GitHub release automatically
+3. Stage the SDK
+    1. On a Google workstation, download the designcompose_m2repo.zip from the release and copy it to `/x20/teams/designcompose/release_staging/`
+    2. Run gmaven_publisher to stage it:
+
+        ```
+        /google/bin/releases/android-devtools/gmaven/publisher/gmaven-publisher stage --gfile /x20/teams/designcompose/release_staging/<the m2repo.zip>
+        ```
+
+    3. The staged release will be available for additional testing (see go/gmaven for more info). You will receive an email explaining how the release can be submitted for publishing and then approval. The release will be published publicly immediately after approval
+4. Update the widget and plugin
+    1. Download the widget and plugin artifacts from the GitHub release onto a system with the Figma Desktop app and unzip them
+    2. Open the Desktop app and open any doc
+    3. Update the Plugin
+        1. Open the **Resources** menu(right of the `T` text menu) and switch to the **Plugins** tab
+        2. Change the dropdown under the search bar to **Development**
+        3. If you already have an entry for the plugin then hover over it, click the `...` menu and click **Remove local version**
+        4. The plugin will now have a **Locate local version** option, click it, then navigate to the `manifest.json` of the plugin artifact you downloaded from GitHub
+        5. Click the `...` menu again and click Publish. Add any release notes and click **Publish new version**
+    4. Do the same as above for the widget (except use the widget menu)
+5. Update the Tutorial
+    1. Open the Tutorial Figma File and create a new branch
+    2. Find each instance of the widget and replace it with the newly published version, matching the settings to the current one
```

Comment:
This is also a huge pain and it would be great to automate this. Since the widget shows the version now we should make sure that it does in fact keep the old version in the doc when we update the widget to a newer version. Maybe opening up the widget in the doc will automatically update to a newer version? We should explore automating this also if not.

Reply:
According to https://www.figma.com/widget-docs/stability-and-updates/#updating-your-widget-safely they're working on it, maybe we should get in touch. But one way to do it would be to have some kind of "export settings" button that generates something that we can paste into the new instance.